easy_my_coop_api/controllers/controllers.py

```diff
@@ -3,11 +3,30 @@
 # License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
 
 
+import re
+
+from odoo import http
 from odoo.http import route
 
 from odoo.addons.base_rest.controllers import main
 
 
+def patch_for_json(path_re):
```

Comment:
@vvrossem @houssine78 I stumbled in this, it might be related to the bug on the test server.